--- apps/ssh/Makefile.in.org	Wed Jun  2 08:37:48 1999
+++ apps/ssh/Makefile.in	Wed Jun  2 08:38:10 1999
@@ -642,8 +642,8 @@
 	 $(DESTDIR)$(mandir)/man8/sshd.old.8
 
 install-exec-hook:
-	chown root $(DESTDIR)$(bindir)/ssh-signer2
-	chmod $(SSH_SIGNER_INSTALL_MODE) $(DESTDIR)$(bindir)/ssh-signer2
+#	chown root $(DESTDIR)$(bindir)/ssh-signer2
+#	chmod $(SSH_SIGNER_INSTALL_MODE) $(DESTDIR)$(bindir)/ssh-signer2
 
 install-data-hook: make-install-dirs generate-host-key install-symlinks
 	-@if test '!' -f $(DESTDIR)$(etcdir)/ssh2/sshd2_config ; then \
